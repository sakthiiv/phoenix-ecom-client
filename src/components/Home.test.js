import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';
describe('Home', () => {
    it('renders correctly', () => {
      const wrapper = shallow(<Home />);
      expect(wrapper).toMatchSnapshot();
      // On the first run of this test, Jest will generate a snapshot file automatically.
    });
});