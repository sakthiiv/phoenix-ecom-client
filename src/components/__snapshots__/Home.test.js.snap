// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <h1>
    Welcome to the Tornadoes Website!
  </h1>
</div>
`;
